client/src/components/Banner/Banner.stories.js

```diff
@@ -0,0 +1,163 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import Banner from "./Banner";
+import InputTextField from "../InputTextField/InputTextField";
+import Button from "../Button/Button";
+
+const handleClick = e => {
+  console.log("Clicked");
+};
+
+storiesOf("Banner", module)
+  .add("Home Banner", () => <Banner
+    bannerType='var(--home-banner)'
+    bannerHeight='380px'
+  />)
+  .add("Home Banner with Welcome text", () =>
+    <Banner bannerType='var(--home-banner)' bannerHeight='380px' bannerText='Welcome,' bannerFontSize='80px' bannerLineHeight='120px'>
+      <div className="Banner__header">
+        <h2>Welcome, Justine</h2>
+      </div>
+    </Banner>
+  )
```

Comment:
The `Banner__header` part should be part of the `Banner` component, so that we can render this like this:

```jsx
<Banner>
  <h2>Welcome, Justine!</h2>
</Banner>
```